activity_browser/actions/exchange/exchange_properties.py

```diff
@@ -0,0 +1,26 @@
+from typing import Any, List
+
+from PySide2 import QtWidgets, QtGui
+
+from activity_browser.actions.base import ABAction, exception_dialogs
+from activity_browser.mod import bw2data as bd
+from activity_browser.ui.widgets.property_editor import PropertyEditor
+
+
+class ExchangeProperties(ABAction):
```

Comment:
Sorry, I can't understand this code on its own. An `Exchange` is very different from an `Activity`, and both the labels and the code itself seem to mix the two. Type hinting isn't really helping here either :)